doc/_index.md

```diff
@@ -1,13 +1,49 @@
 ---
 title: "KCIDB"
-date: 2021-11-18
-draft: false
+date: 2023-08-11
 description: "KernelCI Database service and tools"
 ---
-KCIDB is a package for submitting and querying Linux Kernel CI reports, coming
-from independent CI systems, and for maintaining the service behind that.
+
+[KCIDB](https://github.com/kernelci/kcidb) is a package for submitting and
+querying Linux Kernel CI reports, coming from independent CI systems, and for
+maintaining the service behind that.
 
 See the collected results on [our dashboard](https://kcidb.kernelci.org/).
 Write to [kernelci@lists.linux.dev](mailto:kernelci@lists.linux.dev) if you
 want to start submitting results from your CI system, or if you want to
 receive automatic notifications of arriving results.
+
+The KCIDB [engagement
+reports](https://groups.io/g/kernelci/search?q=%23KCIDB&ct=1) provide more
+details about the latest status of data submitters.
```

Comment:
Perhaps drop this paragraph, since I haven't been sending these for a while, and we moved the maillist?